src/listeners/interactionCreate.ts

```diff
@@ -64,14 +64,27 @@ export default async function (interaction: any) {
     if (rm.has(leaderboards.Configuration[interaction.guildId!].modRole)) {
       const specifiedUser = options.getUser('user')!;
       const specifiedBoard = options.getString('board')!.valueOf();
+      const specifiedOperation = options.getString('operation')!.valueOf();
       const specifiedValue = Math.max(0, options.getNumber('value')!.valueOf())
 
       const ref = db.ref(`/Leaderboard/${specifiedBoard!}/${interaction.guildId}`).child(specifiedUser.id);
-      ref.set(specifiedValue);
+      const newRef = db.ref(`/Leaderboard/${specifiedBoard!}/${interaction.guildId}`).child(specifiedUser.id); //Used to save new value for response message
+      if (specifiedOperation == 'Add') {
+        newRef.set(ref.val() + specifiedValue);

```

Comment:
Not really an issue, this _should_ work, but you can use `set(ServerValue.increment(specifiedValue)`. (same with subtract, just use `-specifiedValue`)

(you'll have to `import { ServerValue } from "firebase-admin/database";`)